### ISSUE TITLE
Bad height calculation

### ISSUE THREAD

**Issue by dynamick** [stylehatch/photoset-grid#66]
Hi,
I've a bad behavior in some cases.

Check this codepen: http://codepen.io/dynamick/pen/vNqbXd

Why it doen't work?

**Comment by ktkaushik** [comment]
One of the things is that you will need to give is the `Layout`

If you don't give layout then it will stick to it's default of **1 column**. You see Photos aligning next to each other because of their CSS properties.

Unlike some libraries, this one **will not** give you a perfect layout unless you specify it to.

**Comment by dynamick** [comment]
Oh, thanks, but the data-layout="23" is given, but the result isn't so good...

However I'll try some other ways to resolve this issues. 
Thanks for your support,
Mick

**Comment by ktkaushik** [comment]
@dynamick - I'm currently using this in my project. The main reason is that the image needs to be 100% loaded for the grid to work.

With that said, it is ideal to have Width and Height attributes added to the `img` element before you call the `photosetGrid` function.

**Comment by ktkaushik** [comment]
@dynamick To keep the whole thing dynamic, i am using the `Image` Element constructor like below - 

```javascript
var newImage = new Image();
newImage.src = img.getAttribute('src');  // img is the Image
newImage.onload = function () {
    img.setAttribute('width', this.naturalWidth);
    img.setAttribute('height', this.naturalHeight);
    index += 1
    setWithAndHeight(index, len);
}
```

**Comment by jonathanmoore** [comment]
This should be fixed with #72
